src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -477,7 +477,7 @@ export class NotebookService extends Disposable implements INotebookService {
 		notebookRegistry.registerNotebookProvider({
 			provider: sqlProvider.providerId,
 			fileExtensions: DEFAULT_NOTEBOOK_FILETYPE,
-			standardKernels: { name: notebookConstants.SQL, displayName: notebookConstants.SQL, connectionProviderIds: [notebookConstants.SQL_CONNECTION_PROVIDER] }
+			standardKernels: { name: notebookConstants.SQL, displayName: notebookConstants.SQL, connectionProviderIds: [notebookConstants.SQL_CONNECTION_PROVIDER, notebookConstants.KUSTO_CONNECTION_PROVIDER] }
```

Comment:
we shouldn't be hardcoding Kusto here. SQL was hardcoded since it was generic (used for all connection providers), but instead, we should be looking at the list of contributed kernel aliases. Plz work with @VasuBhog